packages/web/src/components/basic/SelectedFilters.js

```diff
@@ -42,8 +43,78 @@ class SelectedFilters extends Component {
 	};
 
 	clearValues = () => {
-		const { onClear } = this.props;
-		this.props.clearValues();
+		const {
```

Comment:
@mohdashraf010897 Please refer to the comments on `Vue`.

Reply:
👍